substrings.rb

```diff
@@ -0,0 +1,10 @@
+def substrings(string, substrings)
+  string.downcase!
+  substrings.each_with_object({}) do |substring, occurrences_count_by_substring|
+    substring_occurrences = string.scan(/#{substring.downcase}/).length
+    substring_occurrences.positive? && occurrences_count_by_substring[substring] = substring_occurrences
```

Comment:
although this is correct - try not to set values inside `&&` condtions - for readability - use if
```suggestion
    occurrences_count_by_substring[substring] = substring_occurrences if substring_occurrences.positive?
```

Reply:
done